app/assets/javascripts/tylium/modules/local_auto_save.js

```diff
@@ -1,6 +1,5 @@
 document.addEventListener("turbolinks:load", function() {
-
-  document.querySelectorAll("[data-behavior~=local-auto-save").forEach(function(form) {
+  document.querySelectorAll("[data-behavior~=local-auto-save]").forEach(function(form) {
```

Comment:
Let's use `'` single quotes for this file

Reply:
Fixed!